### ISSUE TITLE
Testing rtMedia with BP Rewrites

### ISSUE THREAD

**Issue by thebengalboy** [rtCamp/rtMedia#1852]
**Description:**
We need to test rtMedia core and all of its addons with the BuddyPress Rewrites 1.0.0-beta2. As this will be a major change, we need to check the compatibility.

**Process to follow**
- Use BuddyPress Rewrites version
- Use Latest WordPress
- Use Latest rtMedia (Released version)
- Use Latest rtMedia Premium (Released version)

Please add the issues on GitHub for each issue observed.

**Comment by thebengalboy** [comment]
@pavanpatil1 BP Rewrites plugin is now available in the WordPress plugin directory(As a separate plugin). Ref: https://buddypress.org/2022/04/bp-rewrites-1-0-1/

Please add this testing to your to-do and you can start it once the current release testing is done. We need to make sure the fixes we have done are working fine and if there are any new changes with the BP rewrites plugin that causes any error/issue.

**Comment by pavanpatil1** [comment]
Hi Mehedi, Tested Both the core and premium plugin with BP-rewrites v1.2.0 https://wordpress.org/plugins/bp-rewrites/. 

- Core plugin is working fine with the latest BP-rewrites version. 
-  On the Premium plugin, It has some compatibility issues with a few addons https://github.com/rtCamp/rtMedia-Pro/labels/BP-rewrites.  no new issue on Premium with the latest BP-rewrites version. 

Closing this issue as task is done.
